velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -93,6 +93,16 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, filter) {
   assertPlanConversion(plan, "SELECT * FROM tmp WHERE c2 < 1000");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, null) {
+  RowVectorPtr vectors = makeRowVector(
+      {makeFlatVector<int32_t>({3, 2}),
+       makeFlatVector<int32_t>(2, nullptr, nullEvery(1))});
```

Comment:
This can be written more clearly using makeNullableFlatVector:

```
makeNullableFlatVector<int32_t>({std::nullopt, std::nullopt})
```

Reply:
fix.